README.md

```diff
@@ -1,4 +1,6 @@
-...
+# Vector Tile Shaver - C++ version
```

Comment:
"C++ version" makes it sound like there's a possible JS version we maintain as well, what do you think about getting rid of this and just having the title be "vtshaver"? @GretaCB